app/src/main/java/com/github/palFinderTeam/palfinder/utils/Filter.kt

```diff
@@ -0,0 +1,60 @@
+package com.github.palFinderTeam.palfinder.utils
+
+import android.widget.Adapter
+import android.widget.Filter
+import android.widget.SearchView
+import com.github.palFinderTeam.palfinder.meetups.MeetUp
+import com.github.palFinderTeam.palfinder.tag.Tag
+import java.util.*
+
+    public class searchedFilter<T>(private val dataSet : List<T>, private val currentDataSet : MutableList<T>, private val callBack : () -> Unit): Filter() {

```

Comment:
It should start with a capital letter (the classname)

Reply:
fixed